packages/designsystem/src/stories/profile/UpdateEmailForm.story.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <Story title="Profile/UpdateEmailForm Form" icon="bi:envelope-at-fill">
+    <Variant title="default">
+      <UpdateEmailForm
+        :user="{
+          id: 1,
+          email: 'contact@locokit.com',
+          profile: 'ADMIN',
+          name: 'Companion Cube',
+        }"
+        @submit="logEvent('submit', $event)"
+      />
+    </Variant>
+  </Story>
+</template>
+
+<script setup lang="ts">
+import { logEvent } from 'histoire/client'
+
+import UpdateEmailForm from '../../components/profile/UpdateEmailForm/UpdateEmailForm.vue'
+</script>
+
+<docs lang="md">
+### UpdateEmailForm
+
+Form which allow an user to update its email if this user is already logged.
```

Comment:
I don't know if it's useful to say the form need to be used when the user is logged.

```suggestion
Form allowing a user to update its email.
```